### ISSUE TITLE
`Image` should support alt text and accessibility label

### ISSUE THREAD

**Issue by waywardmonkeys** [linebender/xilem#1135]
We should be able to set the label on the AccessKit node for an `Image` widget with some alt text.

The easiest version of this would be to have some text on the `Image` widget itself for this.

The harder version, and I think we don't have the infrastructure for it, would be to have a way to say which other node is the label for this image and use `set_labelled_by`.

**Comment by PoignardAzur** [comment]
I'm going to say this is closed by #1528.

As you point out, we don't really have the infrastructure for the `set_labelled_by` part, and we're not planning to add it in the near future. We can re-open this issue if we add it back to the roadmap.
